package.json

```diff
@@ -21,15 +24,15 @@
   "dependencies": {
     "ajv": "^6.10.2",
     "ajv-pack": "^0.3.1",
+    "fs-extra": "^8.1.0",
+    "json-schema-to-typescript": "^7.1.0",
```

Comment:
these need to be (not dev) dependencies to have them included when `npm install --global .`, `npm install json-schema-validators-typescript-interfaces` or `yarn add json-schema-validators-typescript-interfaces`.